test/integration/rules/heading-order/heading-order.json

```diff
@@ -8,6 +8,11 @@
 		["#heading3"],
 		["#heading4"],
 		["#heading5"],
-		["#heading7"]
+		["#heading7"],
+		["#heading8"],
+		["#heading9"],
+		["#heading10"],
```

Comment:
`#heading10` should come after the frame

Reply:
The test results aren't ordered. Nevertheless, probably good to put them in order.